### PR TITLE
offender-events-dev: fix access to topic for hmpps-prisoner-events

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/topic.tf [hunk 1]
```diff
@@ -18,14 +18,31 @@ module "offender_events" {
...
+}
+
+resource "aws_iam_user_policy_attachment" "policy" {
+  policy_arn = module.offender_events.irsa_policy_arn
+  user       = aws_iam_user.hmpps_prisoner_events.name
+}
+
 resource "kubernetes_secret" "offender_events" {
   metadata {
     name      = "offender-events-topic"
```
[the remark this answers is not in the document]
Comment (mattops) [reply namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/topic.tf:38]: Could do, I did name the secret's items with a specific names related to the use.  If this is temporary, not sure it matters too much.

**Review by jakemulley — APPROVED** [review]

**Review by petergphillips — APPROVED** [review]
LGTM

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/topic.tf [hunk 2]
```diff
@@ -18,6 +18,34 @@ module "offender_events" {
...
+resource "aws_iam_access_key" "hmpps_prisoner_events" {
+  user = aws_iam_user.user.name
+}
+
+resource "aws_iam_user_policy_attachment" "hmpps_prisoner_events_policy" {
+  policy_arn = module.offender_events.irsa_policy_arn
+  user       = aws_iam_user.hmpps_prisoner_events.name
+}
+
+resource "kubernetes_secret" " hmpps_prisoner_events" {
```
Comment (jakemulley) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/topic.tf:36]:
```suggestion
resource "kubernetes_secret" "hmpps_prisoner_events" {
```

**Review by jakemulley — DISMISSED** [review]

**Review by jakemulley — APPROVED** [review]

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**